src/applications/vaos/tests/appointment-list/components/CanceledAppointmentsList.unit.spec.js

```diff
@@ -1,62 +1,80 @@
 import React from 'react';
-import MockDate from 'mockdate';
 import { expect } from 'chai';
-import moment from 'moment';
-import environment from '@department-of-veterans-affairs/platform-utilities/environment';
-import { mockFetch, setFetchJSONFailure } from 'platform/testing/unit/helpers';
-import reducers from '../../../redux/reducer';
-import { mockAppointmentInfo } from '../../mocks/helpers';
-import { renderWithStoreAndRouter, getTestDate } from '../../mocks/setup';
+import moment from 'moment-timezone';
+import { mockFetch } from '@department-of-veterans-affairs/platform-testing/helpers';
 import CanceledAppointmentsList from '../../../appointment-list/components/CanceledAppointmentsList';
-import { mockFacilitiesFetchByVersion } from '../../mocks/fetch';
-import {
-  createMockAppointmentByVersion,
-  createMockFacilityByVersion,
-} from '../../mocks/data';
+import { renderWithStoreAndRouter } from '../../mocks/setup';
+import { getVAOSRequestMock } from '../../mocks/v2';
+import { mockVAOSAppointmentsFetch } from '../../mocks/helpers.v2';
 import { APPOINTMENT_STATUS, VIDEO_TYPES } from '../../../utils/constants';
+import { createMockFacilityByVersion } from '../../mocks/data';
 
-const initialState = {
-  featureToggles: {
-    vaOnlineSchedulingCancel: true,
-  },
-};
-
-describe.skip('VAOS Component: CanceledAppointmentsList', () => {
+describe('VAOS Component: CanceledAppointmentsList', () => {
   beforeEach(() => {
     mockFetch();
-    MockDate.set(getTestDate());
-    mockFacilitiesFetchByVersion({ version: 0 });
-  });
-  afterEach(() => {
-    MockDate.reset();
   });
-  it('should show information without facility name', async () => {
-    const startDate = moment.utc();
-    const data = {
+
+  const initialState = {
+    featureToggles: {
+      vaOnlineSchedulingCancel: true,
+      vaOnlineSchedulingVAOSServiceVAAppointments: true,
+    },
+  };
+
+  // This function creates a baseline canceled appointment
+  function mockBaseCanceledAppointment(start) {
+    const appointment = getVAOSRequestMock();
+    appointment.id = '1234';
+    appointment.attributes = {
       id: '1234',
       status: APPOINTMENT_STATUS.cancelled,
       kind: 'clinic',
       clinic: '308',
-      start: startDate.format(),
+      localStartTime: start.format('YYYY-MM-DDTHH:mm:ss.000ZZ'),
```

Comment:
FYI from testing it seemed like `localStartTime` was needed to resolve a valid date time [here](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/vaos/services/appointment/index.js#L392). It's odd that the `start` attribute isn't setting the correct field. I'll dig a little deeper and see if I can figure out why.

Reply:
Ah it's set in the transformers.

https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/vaos/services/appointment/transformers.js#L258

Any reason `localStartTime` is used instead of `start`?